activity.py

```diff
@@ -402,30 +403,27 @@ def guess_resource(self):
             if previous_activity.resource:
                 self.resource = previous_activity.resource
                 self.party = previous_activity.resource.party
+                if not self.party:
+                    self.party = self.on_change_with_party()
         elif self.origin and isinstance(self.origin, ElectronicMail):
-            _, email = parseaddr(self.origin.from_)
+            # parseaddr return first email
+            _, email_from = parseaddr(self.origin.from_)
```

Comment:
Obtenim els mails del "from_" i del "to" del correu electrònic.

Poden ser correus que on s'ha enviat en les dues direccions:

A- "xxx" -> "empleat" 
B-  "empleat" -> "xxx".

En l'exemple de mail ho va enviat un empleat cap un client (xxx).